mlflow/artifacts/__init__.py

```diff
@@ -99,12 +102,47 @@ def download_artifacts(
         raise
 
 
+def _run_artifact_path_corresponds_to_logged_model(artifact_path: str) -> bool:
+    """
+    Does the given artifact path correspond to a logged model name?
+    """
+    return "/" in artifact_path
```

Comment:
This check doesn't correspond to the function name... artifact_path could be a folder and then this returns true, while it could be just an artifact folder instead of a logged model

Reply:
@serena-ruan how would you name this function?

Reply:
Renamed.